package.json

```diff
@@ -42,6 +48,10 @@
     "ember-truth-helpers": "^3.1.1",
     "recast": "^0.18.10"
   },
+  "peerDependencies": {
+    "@appuniversum/ember-appuniversum": "^2.18.0 || ^3.0.0",
```

Comment:
We could also drop 2.18 and bump to 3.0.0 immediately, to not have to cause a breaking change when we would want to use a v3 only feature in  the future.